src/integration/common/headline.js

```diff
@@ -0,0 +1,23 @@
+export default () => {
+  describe('Headline', () => {
```

Comment:
What is headlline? The h1?

Reply:
@greenc05

Reply:
is `Main Heading` better?

Reply:
And the reason we call this headline rather than h1 or headingLevel1 etc?

Reply:
Sorry, commenting at the same time. It's the h1 ya? So IMO I would call it what it is.

Reply:
I try to think of components from a user perspective and not so low-level. A user and possibly a test engineer would call it a headline, title, or main heading. I'd rather go with that

Reply:
Headline could be any heading. Title could be the page title. Main heading is the best option out of those IMO.

Reply:
I agree. Thanks!

Reply:
done!